app/src/main/res/layout/lessons_not_started_chapter_view.xml

```diff
@@ -30,7 +30,7 @@
       android:background="@drawable/chapter_dark_green_bg_with_bright_green_border"
       android:fontFamily="sans-serif"
       android:gravity="center"
-      android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
+      android:importantForAccessibility="yes"
```

Comment:
I think here the values should be `no` instead of yes because the value of `viewModel.chapterPlayState` for this file should be `ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES` which means that the condition will be false, therefore `no`